src/components/RichContenteditable/RichContenteditable.vue

```diff
@@ -176,7 +177,15 @@ export default {
 		 */
 		contenteditable: {
 			type: Boolean,
-			default: true,
+			default: false,
```

Comment:
That is a breaking change.
Default should be true as this is what this component is all about.